src/components/molecules/navSearch/NavSearchSuggestion.tsx

```diff
@@ -0,0 +1,19 @@
+import { Url } from "next/dist/shared/lib/router/router";
+import Link from "next/link";
+import { MouseEventHandler } from "react";
+
+interface NavSearchSuggestionProps {
+  children: React.ReactNode;
+  hint?: React.ReactNode;
+  href: Url;
+  Icon?: React.ReactNode;
+  onClick?: MouseEventHandler<HTMLAnchorElement>;
+}
+
+export const NavSearchSuggestion = ({ children, hint, href, Icon, onClick }: NavSearchSuggestionProps) => (
+  <Link href={href} className="flex items-center gap-2 px-2.5 py-2 rounded-md hocus:bg-surface-ui transition duration-200 group" onClick={onClick}>
```

Comment:
Sanity check: we can use `Link` here because we are always starting a new search context? Therefor we should never need to use `LinkWithQuery` right?